### ISSUE TITLE
Trademark usage of Spinnaker

### ISSUE THREAD

**Issue by dnilasor** [cdfoundation/foundation#215]
Hey, can someone look into the trademark usage demonstrated in https://marketplace.redhat.com/en-us/products/spinnaker and reach out to OpsMX about usage of the Spinnaker mark? Per a prior conversation with @caniszczyk and @brianwarner, Armory has updated our product name and we're nearly done the process of updating all the assets on our website (started with the most prominent) to comply with the guidelines from the LF https://www.linuxfoundation.org/trademark-usage/. So, now we're extra aware of those guidelines. Thanks!

**Comment by caniszczyk** [comment]
@dnilasor I notified @tracymiranda who now is the CDF lead and is working to rectify the situation and kickstart a conformance working group for Spinnaker + put together branding guidelines.

**Comment by dnilasor** [comment]
Hi. Any movement on this? Does the Linux Foundation actually own the trademark for Spinnaker at this juncture? @tracymiranda can you please look into this or refer us elsewhere? We are still unable to use the word Spinnaker in Google advertisements because of a conflict with Spinnaker Support, who owns a relevant mark. We are unable to resolve this on our own as we are not the mark owner. Our competitors are still able to use the mark because of the timing of the complaint.

**Comment by caniszczyk** [comment]
for sure the LF owns the mark, I just looked it up in USPTO
<img width="1166" alt="Screen Shot 2021-02-08 at 3 10 02 PM" src="https://user-images.githubusercontent.com/63777/107281834-d2513580-6a1f-11eb-9951-3423a7581ef3.png"> along with it being here: https://www.linuxfoundation.org/trademarks

@tracymiranda leads CDF so should be working to resolve this issue with you and the community

**Comment by dnilasor** [comment]
Great, looks like the filing was finalize and approved in November, since the last time I pinged on this. Thanks for the update @caniszczyk. I am reaching out to the complainant at Spinnaker Support now, Michelle, whose contact information I got from Google Ads. If there are any additional disputes about this, I will likely need support from the CDF/LF since you are the owner of the mark and have authorized to use it (they may want verification). 

I'll reach out to Tracy in that case and request expedient attention. Thanks!
